.github/workflows/autotools.yml

```diff
@@ -159,6 +159,7 @@ jobs:
           test/command_line/run-test.rb --groonga-install-prefix=$PWD/install
       - name: "Test: stdio"
         run: |
+          MEACAB_DIC=naist-jdic \
```

Comment:
Hmm. If we add a test that needs ipadic, do we need to rewrite this approach?

Reply:
I should think of another approach about this.
What do you think of passing used dictionaries using grntest's option?
```
grntest --mecab-dictionaries=ipadic,naist-jdic
```

Reply:
In general, we can't use multiple dictionaries with `TokenMecab` because it always uses the default dictionary.

So `--mecab-dictionaries=ipadic,naist-jdic` will not work.

Reply:
Ah, I misunderstood. Our code only checks whether `MECAB_DIC` is set, not what its value actually is. Since we can’t know which dictionary it points to, we have no way to know which dictionary-specific tests to skip. That means if we want to add tests for `ipadic`, we still can’t selectively disable the tests for other dictionaries.

So we should pass one dictionary information. I will add a option to `grntest`.
```
grntest --mecab-dictionaries=ipadic
```

Reply:
If we want to use the approach, we can use `--mecab-dictionary=ipadic` because we can't use multiple dictionaries at once.
If we want to use the environment variable approach, we need to use `MECAB_DICTIONARY_NAIST_JDIC` or something.

Reply:
fix: [ba714ab](https://github.com/groonga/groonga/pull/2331/commits/ba714ab156c71dcb50b399cd8beb33a41d831aad) I took the environment variable approach because adding MeCab's option to grntest is

I chose the environment‐variable approach because adding a MeCab‐specific flag to grntest felt out of place. I though the grntest options shouldn't be used for handling individual test cases.